test/kixi/integration/upload_test.clj

```diff
@@ -20,45 +27,186 @@
                                  :sharing {:read [uid]
                                            :use [uid]}}})
 
-
 (defn setup-schema
-  [all-tests]
+  []
   (let [r (post-spec-and-wait uid irrelevant-schema)]
     (if (= 202 (:status r))
       (reset! irrelevant-schema-id (extract-id r))
-      (throw (Exception. "Couldn't post irrelevant-schema"))))
-  (all-tests))
-
-(use-fixtures :once cycle-system-fixture setup-schema)
+      (throw (Exception. "Couldn't post irrelevant-schema")))))
 
 (deftest round-trip-files
-  (let [r (post-file uid
-                     "./test-resources/metadata-one-valid.csv"
-                     @irrelevant-schema-id)]
-    (is (= 201
-           (:status r))
-        (str "Reason: " (parse-json (:body r))))
-    (when-let [locat (get-in r [:headers "Location"])]
-      (is (files-match?
-           "./test-resources/metadata-one-valid.csv"
-           (dload-file uid locat)))))
-  (let [r (post-file uid
-                     "./test-resources/metadata-12MB-valid.csv"
-                     @irrelevant-schema-id)]
-    (is (= 201
-           (:status r))
-        (str "Reason: " (parse-json (:body r))))
-    (when-let [locat (get-in r [:headers "Location"])]
-      (is (files-match?
-           "./test-resources/metadata-12MB-valid.csv"
-           (dload-file uid locat)))))
-  (let [r (post-file uid
-                     "./test-resources/metadata-344MB-valid.csv"
-                     @irrelevant-schema-id)]
-    (is (= 201
-           (:status r))
-        (str "Reason: " (parse-json (:body r))))
-    (when-let [locat (get-in r [:headers "Location"])]
-      (is (files-match?
-           "./test-resources/metadata-344MB-valid.csv"
-           (dload-file uid locat))))))
+  (try 
+    (repl/start)
```

Comment:
originally the namespace was for using via the repl... the name seems a bit out of date now :)